index.html

```diff
@@ -9,103 +9,197 @@
+          <button class="btn" >Book now</button>
+          <figcaption class="content-home-img-fig">Maldives</figcaption>
+        </figure>
+        <figure>
+          <img class="" src="img/Rectangle 53.png" alt="Spain">
+          <figcaption class="content-home-img-fig ">Spain</figcaption>
+        </figure>
+        <figure class="figcap-768">
+          <img  class="figcap" src="img/Rectangle 54.png" alt="Norway">
+          <figcaption class="content-home-img-fig figcap"> Norway</figcaption>
+        </figure>
+        <figure class="figcap-1220">
+          <img class="img-button figcap figcap-m" src="img/Rectangle 55.png" alt="Island">
+          <figcaption class="content-home-img-fig figcap figcap-m ">Island</figcaption>
+        </figure>
+        <button class="content-button-imgs" id="content-button-imgs"><img class="button-img-s-s " src="img/Polygon 2.svg" alt=""></button>
+      </div>
+    </div>
+  </div>
+
+</section>
+<!--Sign up-->
+<section>
+<aside class="profile-bg">
+  <div class="profile" >
+  <button class="profile-user"><img src="img/user.png" alt=""></button>
+  <div class="profile-p-b">
+    <h3>Sign up and get 20% off your first booking</h3>
+    <button class="profile-sing-button">Sign up</button>
+  </div>
+  <button class="profile-clouse"><img src="img/Vector.svg" alt=""></button>
+  </div>
+</aside>
 
-      <ul class="otdelni">
-        <li class="bold footer-text">About</li>
-        <li class="footer-text">How Triphouse work</li>
-        <li class="footer-text" >Careers</li>
-        <li class="footer-text" >Privacy</li>
-        <li class="footer-text" >Terms</li>
-      </ul>
-      <ul class="otdelni">
-        <li class="bold footer-text">Property types</li>
-        <li class="footer-text" >Guest houses</li>
-        <li class="footer-text" >Hotels</li>
-        <li class="footer-text" >Apartments</li>
-        <li class="footer-text" >Villas</li>
-        <li class="footer-text" >Holiday homes</li>
-        <li class="footer-text" >Hostels</li>
-      </ul>
-      <ul class="otdelni">
-        <li class="bold footer-text">Support</li>
-        <li class="footer-text" >Contact Customer Service</li>
-        <li class="footer-text" >FAQ</li>
-      </ul>
+  <aside class="mob-profile-bg">
+    <div class="mob-profile" >
+    <button class="mob-profile-user"><img class="mob-profile-user-i" src="img/Vector.png" alt=""></button>
+    <img class="tripHouseMob-img" src="img/tripHouseMob.svg" alt="tripHouseMob">
+    <div class="mob-profile-h-a">
+      <h3 class="mob-profile-h" >Sign up in the app and get 20% off your first booking</h3>
+      <a class="mob-profile-a">Install</a>
     </div>
-    <div class="triphouse">
-      <p>&#169;2020 Triphouse, Inc.All rights reserved</p>
     </div>
-  </section>
+  </aside>
+</section>
+<!--footer-->
+<footer>
+  <div class="footer" >
+    <div class="fotter-width">
+      <div class="footer-about" >
+        <ul class="otdelni">
```

Comment:
try to use more abstract names for classes, for example footer-column in this place.